azkaban-sql/src/sql/create.execution_flows.sql

```diff
@@ -13,10 +13,11 @@ CREATE TABLE execution_flows (
 	flow_data LONGBLOB,
 	executor_id INT DEFAULT NULL,
 	PRIMARY KEY (exec_id)
-);
+)
+ENGINE=InnoDB DEFAULT CHARSET=latin1;
```

Comment:
Is there a concern with moving charset to utf8? I remember a user asked for UTF8 support to store international characters ( maybe in project names?).